front/components/SongCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Card, { CardBorderRadius } from './Card';
+import { VStack, Text, Image, Pressable } from 'native-base';
+import { useNavigation } from "@react-navigation/core";
+type SongCardProps = {
+	albumCover: string;
+	songTitle: string;
+	artistName: string;
+	songId: number
+}
+
+const SongCard = (props: SongCardProps) => {
+	const { albumCover, songTitle, artistName, songId } = props;
```

Comment:
note: you can use this notation directly in the function declaration if you don't use props later